lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -82,7 +88,7 @@ def last_committed_at
 
     def analyze_last_committed_at(targets)
```

Comment:
[suggestion] How about adding an additional message to trace?

```suggestion
    def analyze_last_committed_at(targets)
      trace_writer.message "Analyzing last commit time..."

```

Reply:
I agree. Plus, it would be good for us to know the duration of whole calculation by wrapping `targets.each do ...` with `trace_writer.message`.

```ruby
trace_writer.message "Analyzing last commit time..." do
  targets.each do |target|
    ...
  end
end

Reply:
fix: b9cc4cd0f9e2c120b53f91ff71aa61369d2d2f7e